coalib/bearlib/languages/definitions/Verilog.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Verilog:
+    aliases = 'verilog',
+    extensions = '.v',
+    comment_delimiters = '//',
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'"': '"'}
+    multiline_string_delimiters = {}
```

Comment:
I don't think u need to include `multiline_string_delimiters` if there isn't any.